rllib/models/modelv2.py

```diff
@@ -318,6 +319,24 @@ def is_time_major(self) -> bool:
         """
         return self.time_major is True
 
+    @PublicAPI
+    def update_view_requirements_from_init_state(self):
```

Comment:
I don't think we should be exposing this as a public API of model. Instead, why not have the policies add the state view requirements internally, without mutating or otherwise requiring the model to do anything.

Reply:
+1

Reply:
done